packages/workspace/src/generators/convert-to-monorepo/convert-to-monorepo.from-nx-init.spec.ts

```diff
@@ -0,0 +1,34 @@
+import 'nx/src/internal-testing-utils/mock-project-graph';
```

Comment:
Had to extract this to another file, because it was causing `preset.spec.ts` to fail on agents for some reason, without any useful failures/errors printed. It could not be reproduced locally.